### PR TITLE
Update Wagmi guide and tutorial

### LINKED ISSUE #1668: Improve Wagmi guide
The Wagmi SDK guide requires several updates and corrections.

### PR DESCRIPTION
**Pull request by alexandratran**
# Description

This PR updates the Wagmi guide and tutorial to be more consistent with each other, using the `create wagmi` CLI and omitting options that we don't recommend configuring.

## Issue(s) fixed

Fixes #1668

## Preview

- Wagmi guide: https://metamask-docs-aigke8hac-consensys-ddffed67.vercel.app/wallet/connect/3rd-party-libraries/wagmi/
- Wagmi tutorial: https://metamask-docs-aigke8hac-consensys-ddffed67.vercel.app/wallet/tutorials/react-dapp-sdk-wagmi/

## Checklist

Complete this checklist before merging your PR:

- [ ] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

File: wallet/connect/3rd-party-libraries/wagmi.md [hunk 1]
```diff
@@ -15,61 +15,75 @@ the MetaMask browser extension and MetaMask Mobile.
...
+- Basic knowledge of [React](https://react.dev/)
 
-## Configure the MetaMask connector
+## Steps
 
-In your Wagmi project, configure the MetaMask connector:
+### 1. Set up the project
 
-```javascript
+If you don't have an existing Wagmi project, set up a new project using Wagmi's `create wagmi` command
```
Comment (bgravenorst) [inline wallet/connect/3rd-party-libraries/wagmi.md:29]: I assume this must be done in your project folder? It's probably obvious to developers, so you can decide whether you want to clarify this or not.

EDIT: Reading below, this actually creates a project folder, so probably worth clarifying?

File: wallet/connect/3rd-party-libraries/wagmi.md [hunk 2]
```diff
@@ -15,61 +15,75 @@ the MetaMask browser extension and MetaMask Mobile.
...
+For example, use `mmsdk-wagmi`.
+Once your project is created, navigate into it and install the node module dependencies:
+
+```bash
+cd mmsdk-wagmi && npm install
+```
+
+### 2. Configure the MetaMask connector
+
+In `wagmi.ts`, configure the MetaMask connector with any [parameters](https://wagmi.sh/core/api/connectors/metaMask#parameters).
```
Comment (bgravenorst) [inline wallet/connect/3rd-party-libraries/wagmi.md:46]: any = the required?

File: wallet/connect/3rd-party-libraries/wagmi.md [hunk 3]
```diff
@@ -15,61 +15,75 @@ the MetaMask browser extension and MetaMask Mobile.
...
-option to [make read-only requests](../../how-to/make-read-only-requests.md) using the Infura API.
-You can set your Infura API key in environment variables:
-
-```env
-VITE_INFURA_API_KEY=<YOUR-API-KEY>
-```
-
-## Benefits of using the Infura API with Wagmi
+:::note
+By default, if the EIP-6963 MetaMask injected provider is detected, this connector will replace it.
```
Comment (bgravenorst) [inline wallet/connect/3rd-party-libraries/wagmi.md:72]: Should we link to https://eips.ethereum.org/EIPS/eip-6963

File: wallet/tutorials/react-dapp-sdk-wagmi.md [hunk 4]
```diff
@@ -42,6 +42,8 @@ custom connect button.
 
 ### 1. Set up the project
 
+Set up your project using Wagmi's `create wagmi` command with the `vite-react` template:
```
Comment (bgravenorst) [inline wallet/tutorials/react-dapp-sdk-wagmi.md:45]: Similar comment as the previous

**Review by bgravenorst — APPROVED** [review]
Minor feedback.

File: wallet/connect/3rd-party-libraries/wagmi.md (hunk 2 above)
Comment (alexandratran) [reply wallet/connect/3rd-party-libraries/wagmi.md:46]: These are more like options, so I think "any" is appropriate, but the team prefers using Wagmi's term "parameters."

File: wallet/connect/3rd-party-libraries/wagmi.md (hunk 1 above)
Comment (alexandratran) [reply wallet/connect/3rd-party-libraries/wagmi.md]: It's pretty standard for a "create" CLI tool to create the project directory itself (e.g. Create React App), so I think most developers will already know. However, I tried clarifying the sentence a little.

**Review by bgravenorst — APPROVED** [review]
lgtm

**Pull request merged**